utils/to/arn.go

```diff
@@ -72,6 +72,19 @@ func AwsRegionAccountFromContext(ctx context.Context) (*string, *string) {
 	return &region, &account
 }
 
+func AwsRegionAccountLambdaNameFromContext(ctx context.Context) (string, string, string) {
+	arn, err := LambdaArnFromContext(ctx)
+	if err != nil {
+		return "", "", ""
+	}
+
+	region, account, resource := ArnRegionAccountResource(arn)
+	// function:<lambda name>
+	resourceParts := strings.SplitN(strings.ToLower(resource), ":", 2)
+
+	return region, account, resourceParts[1]
```

Comment:
should check that this is at least 2 long before grabbing the second item here. If resource paths is not long enough then return empty string